index.html

```diff
@@ -73,10 +73,69 @@ <h1 class="worktext">My Recent Works  </h1>
             <li class="langc">javascript</li>
             <li class="langd">ruby</li>
         </ul>
-        <button class="buttonw"><ins class="seep">See Project</ins></button>
+        <button class="buttonw" type="button"><ins class="seep">See Project</ins></button>
     </div>
 </div>
 <!--End Recent of works section-->
+<section class="projectcards">
```

Comment:
- [ ] [OPTIONAL] on mobile view all the projects disappeared kindly consider fixing this so all your project are present in mobile view also

![Screenshot_20220903_223034](https://user-images.githubusercontent.com/25227022/188288294-a8f4fbc0-16cf-4ab5-a72e-895136ae0bfc.png)